### DISCUSSION TITLE
migrating from create-react-app

### DISCUSSION THREAD

**Discussion by nikinbaidar** [vercel/next.js#64361] [Help]
### Summary

Hi folks! I am trying to migrate a website I initially  built using `create-react-app` to next. I have pretty much followed the instructions I found [here](https://nextjs.org/docs/app/building-your-application/upgrading/from-create-react-app) and the website compiles all right. Except I have two concerns:

1. I get this error on my build

```
 Error: Page "/[[...slug]]/page" is missing param "/[object%20Object]" in "generateStaticParams()", which is required with "output: export" config.
    at DevServer.renderToResponseWithComponentsImpl (/home/nikin/projects/notesstudio/node_modules/next/dist/server/base-server.js:1032:27) {
  page: '/[object%20Object]'
}
 ⨯ Error: Page "/[[...slug]]/page" is missing param "/[object%20Object]" in "generateStaticParams()", which is required with "output: export" config.
    at DevServer.renderToResponseWithComponentsImpl (/home/nikin/projects/notesstudio/node_modules/next/dist/server/base-server.js:1032:27) {
  page: '/[object%20Object]'
}
}
```
This is my `app/[[...slug]]/page.js`
```
import '../../index.css'
import { ClientOnly } from './client'
 
export function generateStaticParams() {
  return [{ slug: [''] }]
}
 
export default function Page() {
  return <ClientOnly />
}

```

Looking at the error, I see I need to change the object that `generateStaticParams()` returns. But I don't know what exactly I must do. 

2. The second concern is that none of images are rendered. I am making the imports as per the docs

```
import picture from './images/file.webp';
```

**Comment by raulvictorrosa** [comment]
I'm having the same problem.

**Comment by vinay-nb** [comment]
I am facing the same issue, is there any solution, open for suggestions

**Comment by armordog** [comment]
I'm from the future and this guide still does not work.
